otx/algorithms/common/adapters/mmcv/tasks/stage.py

```diff
@@ -499,19 +512,20 @@ def read_label_schema(ckpt_path, name_only=True, file_name="label_schema.json"):
             all_classes = []
         return all_classes
 
+    # pylint: disable=unused-argument
     @staticmethod
     def set_inference_progress_callback(model, cfg):
-        # InferenceProgressCallback (Time Monitor enable into Infer task)
+        """Inferenceprogresscallback (Time Monitor enable into Infer task)."""
         time_monitor = None
         if cfg.get("custom_hooks", None):
             time_monitor = [hook.time_monitor for hook in cfg.custom_hooks if hook.type == "OTXProgressHook"]
             time_monitor = time_monitor[0] if time_monitor else None
         if time_monitor is not None:
 
-            def pre_hook(module, input):
+            def pre_hook(*args):
```

Comment:
```suggestion
            def pre_hook(*args, **kwargs):
```
Isn't kwargs needed?